### ISSUE TITLE
Regarding Training Methods and Results

### ISSUE THREAD

**Issue by JChenenene** [LabShuHangGU/CCA#7]
I trained according to the commands provided in the code, with lambda set to 15 and the command used being
CUDA_VISIBLE_DEVICES='0' python -u ./train.py -d [path of training dataset] \
-lr 1e-4 --cuda --beta 20 --epochs 65 --lr_epoch 56 --batch-size 8 \
--save_path [path for storing the checkpoints] --save \
--checkpoint [path of the pretrained checkpoint]，
The training set consisted of 300K images extracted from openImage, and the final CCA loss was -37.052. The final result of the training did not meet expectations. The test result for the kodak dataset was average PSNR: 29.73dB， average_MS-SSIM: 12.2062，average_Bit-rate: 0.147 bpp， The effect is very poor, may I ask what the reason is?
